SW-data_analysis-20220111.do

```diff
@@ -82,7 +82,7 @@ replace PHQ_CAT = "Severe depression" if PHQ > 19
 
 forvalues y = 2019/2020{
 	gen depressed_`y' = .
-	replace depressed_`y' = 1 if PHQ_`y' >= 8
+	replace depressed_`y' = 1 if PHQ_`y' >= 8 & PHQ_`y' < . 

```

Comment:
I noticed that ``PHQ_y`` might have missing values. If you are not coding  
``depressed_y = 1  if PHQ_y== .``
on purpose
this need to add condition ``PHQ_y >= 8 & PHQ_y < . `` to avoid this.